decidim-admin/app/forms/decidim/admin/organization_form.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+module Decidim
+  module Admin
+    # A form object used to update the current organization from the admin
+    # dashboard.
+    #
+    class OrganizationForm < Rectify::Form
+      include TranslatableAttributes
+
+      translatable_attribute :description, String
```

Comment:
Can you move this after `attribute :name, String`? This way we have everything grouped.